{
	"id": "CVE-2022-1508",
	"description": "An out-of-bounds read flaw was found in the Linux kernelâ€™s io_uring module in the way a user triggers the io_read() function with some special parameters. This flaw allows a local user to read some memory out of bounds.",
	"scope": "local",
	"packages": [
		{
			"name": "linux",
			"status": "resolved",
			"urgency": "not yet assigned",
			"fixed_version": "5.15.3-1",
			"repository": [
				{
					"name": "bookworm",
					"version": "6.1.66-1"
				},
				{
					"name": "bookworm-security",
					"version": "6.1.52-1"
				}
			]
		}
	]
}
